src/concepts/glean/debug_ping_view.md

```diff
@@ -0,0 +1,85 @@
+# Using the Glean debug ping view
+
+<!-- toc -->
+
+## What is this good for?
+
+This enables you to easily see in real-time what data your mobile application is sending through [Glean](glean.md).
+
+This data is what actually arrives in our data pipeline, shown in a web interface that is automatically updated when new data arrives.
+
+## What setup is needed for applications?
+
+You can use the debug view for all our mobile applications that use Glean (and enable it), including those installed from the app store. To enable this you need to run a command in adb that tags the outgoing data as "debug data". You will provide a debug tag, which makes it easier to identify your device in the web interface.
+
+```
+adb shell am start -n <application-id>/mozilla.components.service.glean.debug.GleanDebugActivity \
+  --ez logPings true \
+  --es sendPing baseline \
+  --es tagPings my-debug-tag
+```
+
+_my-debug-tag_ is what will help you identify your data in the web interface, while `<application-id>`  is the application identifier as declared in the [manifest](https://developer.android.com/studio/build/application-id) (e.g. `org.mozilla.reference.browser`). The debug commands are documented in more detail [in the Glean README](https://github.com/mozilla-mobile/android-components/blob/master/components/service/glean/README.md#debugging-products-using-glean).
+
+### Supported applications
+
+As for now, the following application ids are supported:
+
+*   `org.mozilla.fenix`
+*   `org.mozilla.reference.browser`
+*   `org.mozilla.samples.glean`
+
+## Where can I see the data?
+
+The data is provided in [this web interface](https://debug-ping-preview.firebaseapp.com/). It lists all recently active devices and updates automatically. You can use your debug identifier to quickly identify your own testing data.
+
+Any data sent from a mobile device usually shows up within 10 seconds, updating the pages automatically.
+
+## Can you give me an example?
```

Comment:
Any chance we could link or embed [this](https://github.com/mozilla-mobile/android-components/tree/master/components/service/glean#important-gleandebugactivity-notes) from the Glean SDK docs? We had a couple of bugs filed that made us notice that the supported flows were not clear :(